src/SAML2/Compat/Ssp/Container.php

```diff
@@ -114,4 +120,42 @@ public function writeFile(string $filename, string $data, int $mode = null): voi
         /** @psalm-suppress UndefinedClass */
         System::writeFile($filename, $data, $mode);
     }
+
+
+    /**
+     * @inheritDoc
+     */
+    public function registerExtensionHandler(string $class): void
+    {
+        Assert::subclassOf($class, AbstractXMLElement::class);
+
+        if ($class instanceof CustomIdentifierInterface) {
```

Comment:
Unfortunately this wont work for class strings.. I had experienced and fixed this issue before, but it was probably force-pushed away..
The only way to achieve this is to `in_array(CustomIdentifierInterface::class, class_implements($class))`